src/components/balance/Address.vue

```diff
@@ -1,85 +1,35 @@
 <template>
-  <div
-    class="
-      tw-relative tw-z-0 tw-inline-flex tw-shadow tw-rounded-lg tw-bg-white
-      dark:tw-bg-darkGray-800
-    "
-  >
-    <div
-      class="
-        tw-relative tw-inline-flex tw-items-center tw-py-4 tw-px-1
-        sm:tw-py-5 sm:tw-px-2
-        tw-rounded-l-lg tw-flex-1 tw-text-left
-      "
-    >
+  <div class="tw-inline-flex">
+    <div class="tw-relative tw-inline-flex tw-items-center tw-rounded-l-lg tw-flex-1 tw-text-left">
       <div class="tw-flex tw-items-center">
-        <div
-          class="
-            tw-h-11 tw-w-11
-            sm:tw-h-12 sm:tw-w-12
-            tw-rounded-full tw-overflow-hidden
-            sm:tw-border
-            tw-border-gray-100 tw-mx-2
-            sm:tw-mx-3
-          "
-        >
-          <icon-base class="tw-h-full tw-w-full" viewBox="0 0 64 64">
-            <icon-account-sample />
-          </icon-base>
+        <div class="tw-h-11 tw-w-11 sm:tw-h-12 sm:tw-w-12 tw-overflow-hidden tw-mx-2 sm:tw-mx-3">
+          <div v-if="format === 'H160'">
+            <img width="80" src="~assets/img/metamask.png" />
```

Comment:
Instead of using the MetaMask icon, please use the Ethereum icon
![2000px-Ethereum_logo svg](https://user-images.githubusercontent.com/40356749/141680574-088a9abc-8aef-4242-9dd2-bcc8f8080bba.png)